code/controllers/subsystem/parallax.dm

```diff
@@ -57,3 +65,32 @@ SUBSYSTEM_DEF(parallax)
 		if (MC_TICK_CHECK)
 			return
 	currentrun = null
+
+/// Generate a random layer for parallax
+/datum/controller/subsystem/parallax/proc/set_random_parallax_layer(picked_parallax)
+	if(picked_parallax == PARALLAX_NONE)
+		return
+
+	random_layer = new picked_parallax(null, null, TRUE) //rip no named params, but we set template to TRUE so we don't del without a hud owner
```

Comment:
```suggestion
	random_layer = new picked_parallax(null, null, /* template = */ TRUE) //rip no named params, but we set template to TRUE so we don't del without a hud owner
```
Mothblocks has suggested this pattern before where named args are not possible, would be nice to add all 3